manifests/m/Microsoft/AzureCLI/2.60.0/Microsoft.AzureCLI.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.AzureCLI
+PackageVersion: 2.60.0
+Installers:
+- Architecture: x86
+  InstallerType: msi
+  InstallerUrl: https://azcliprod.azureedge.net/msi/azure-cli-2.60.0.msi
+  InstallerSha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+- Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://azcliprod.azureedge.net/msi/azure-cli-2.60.0-x64.msi
```

Comment:
URL has changed to https://azcliprod.blob.core.windows.net/msi/azure-cli-2.60.0-x64.msi